src/components/VaultStats.tsx

```diff
@@ -114,14 +114,14 @@ const VaultStats = ({ isModifying, isDeposit }: { isModifying: boolean; isDeposi
                             <MainLabel>OD Debt</MainLabel>
                             <RowWrapper>
                                 <div>
-                                    <MainValue>{formatWithCommas(totalDebt)}</MainValue>
-                                    <MainChange>${formatWithCommas(totalDebtInUSD)}</MainChange>
+                                    <MainValue>{formatWithCommas(totalDebt, 2, 2)}</MainValue>
```

Comment:
Remove the minimum decimals for `totalDebt`

Reply:
- collateral, newCollateral, totalDebt  now don't have min decimals, max is 8 as a default in a format function. So if a whole number it will be without decimals, but if not it will show maximum 8
- collateralInUSD, totalDebtInUSD now has minimum and max two decimals ("Currency amounts should always have only 2 decimals eg "$1.20"")

Reply:
Also formatted with 2 decimals `liquidationPrice` and `newLiquidationPrice`

<img width="893" alt="Screenshot 2023-08-25 at 1 45 19 PM" src="https://github.com/UseKeyp/od-app/assets/36742189/96f60993-ed8c-4e86-b233-a1bdd304f4b3">